### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by hectorsector**
Couple of quick fixes to the README.

### CODE REVIEW

**Comment by hectorsector** [comment]
> [...] a payload signature. Looking Glass reads the output of a previous GitHub action. To use [...]

When I read this sentence, I'm expecting to also learn about what Looking Glass does with the output of the action. Maybe we can add something about reporting using the specified GitHub feature?

> `filename: "can be empty if no file is associated with the report",`

In the second example, could we maybe just leave the filename off? We can include the "can be empty..." copy in the first example. 

> ```
> msg:
>        "# something was incorrect\n**expected:** some correct answer\n**got:** some incorrect answewr",
>    },
> ```

This is actually a pretty common pattern, message/expected/got. Should we support this?

**Comment by mattdavis0351** [comment]
>When I read this sentence, I'm expecting to also learn about what Looking Glass does with the output of the action. Maybe we can add something about reporting using the specified GitHub feature?

Sure, the details weren't my initial mindspace when writing this up, but if you have an idea let's add it and see!

>In the second example, could we maybe just leave the filename off? We can include the "can be empty..." copy in the first example.

We can just leave it blank since it's shown as not required in the table of payload values.

>This is actually a pretty common pattern, message/expected/got. Should we support this?

I like this idea, what if `msg` was

```
msg:{
    expected: "something",
    got: "this thing"
}
```

**Comment by mattdavis0351** [comment]
I'm also not sold on this payload being an array.  I don't think Looking Glass should support more than one report for initial release.

We could keep it at an array an only focus on the first item in the array, that way we can extend this easier in the future, but we may want to add some sort of key to the payload that informs Looking Glass what kind of report it's getting

- hands on lab
- some maintenance thing
- some other sort of feedback
- etc

**Comment by hectorsector** [comment]
How about the following, ideally if we could support just `msg: "some message"` AND the longform:

```
msg:{
    context: "general message to use",
    expected: "something",
    got: "this thing"
}
```

> I'm also not sold on this payload being an array.

I agree, but this is only relevant to the action itself right? What are the alternatives? 

> we may want to add some sort of key to the payload that informs Looking Glass what kind of report it's getting

I'm definitely focused on the hands-on lab so I haven't been thinking about how this impacts other potential uses.

File: README.md [hunk 1]
```diff
@@ -15,24 +15,24 @@ Looking Glass is a reporting agent that provides feedback to users based on a pa
       msg: "some string that contains a message",
     },
     {
-      filename: "can be empty if no file is associated with the report",
+      filename: "",
       isCorrect: false,
       display_type: "issues",
       level: "fatal",
       msg:
```
Comment (hectorsector) [inline README.md:22]: Potentially:
```
{ ...
msg: "the message",
error: {
 expected: "the expected string",
 got: "the gotten string"
 }
... }
```

Comment (mattdavis0351) [reply README.md:22]: i like this

**Review by mattdavis0351 — APPROVED** [review]

**Pull request merged**